CHANGELOG.md

```diff
@@ -150,28 +150,39 @@
 - Expand support for DataFrames with no rows in `pd.pivot_table` and `DataFrame.pivot_table`.
 - Added support for `inplace` parameter in `DataFrame.sort_index` and `Series.sort_index`.
 
-
 ## 1.19.0 (2024-06-25)
 
 ### Snowpark Python API Updates
 
+#### Improvements
+
+- Added open telemetry tracing on UDxF functions in snowpark.
+- Added open telemetry tracing on stored procedure registration in snowpark.
+
 #### New Features
 
 - Added support for `to_boolean` function.
-- Added documentation pages for Index and its APIs.
 
 #### Bug Fixes
 
 - Fixed a bug where python stored procedure with table return type fails when run in a task.
 - Fixed a bug where df.dropna fails due to `RecursionError: maximum recursion depth exceeded` when the DataFrame has more than 500 columns.
 - Fixed a bug where `AsyncJob.result("no_result")` doesn't wait for the query to finish execution.
-
+- Fixed a bug regarding precision loss when converting to Snowpark pandas `DataFrame` or `Series` with `dtype=np.uint64`.
```

Comment:
@sfc-gh-mrojas Is this a part of the PR? The changes in changelog are not related to your code change.

Reply:
No that should be an error in when I merged the readme let me check

Reply:
@sfc-gh-yixie I just resynced the changelog file